<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 223, 223);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_admin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_login">
    <widget class="QGroupBox" name="box_login">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>431</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 203, 203);</string>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <widget class="QPushButton" name="button_login">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_password">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_username">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_login_status">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>430</y>
       <width>281</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>[+] Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAdminTMain">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>Main Page</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_db__interface">
    <widget class="QPushButton" name="button_view_customer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>View Customer List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_view_key_customer">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>View Key Customer List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_view_purchases">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>View Purchases</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_edit_customer_db">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>Edit Customer Database</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_view_database">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>971</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
    background-color:rgb(0, 145, 145);
}
QHeaderView::section:checked
{
    background-color: rgb(0, 203, 203);
}
QTableView {
   background-color:rgb(0, 185, 185);
}</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_edit_database">
    <widget class="QGroupBox" name="groupBox_information">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>621</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 203, 203);</string>
     </property>
     <property name="title">
      <string>Customer Information</string>
     </property>
     <widget class="QLabel" name="label_address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>68</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_website">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>68</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Website</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_interest_level">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Interest Level</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_do_not_contact">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Do Not Contact</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_key_customer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Key Customer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pamphlet_sent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Pamphlet Sent</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>68</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_name">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>471</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_address">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>471</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_website">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>471</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_interest_level">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>471</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_do_not_contact">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_pamphlet_sent">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField_key_customer">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_options">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>371</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 203, 203);</string>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QPushButton" name="button_add_customer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_update_customer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Update Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_customer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Delete Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 185, 185);</string>
      </property>
      <property name="text">
       <string>Back to Viewing Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_edit_database">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>1001</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
    background-color:rgb(0, 145, 145);
}
QHeaderView::section:checked
{
    background-color: rgb(0, 203, 203);
}
QTableView {
   background-color:rgb(0, 185, 185);
}</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
